Sources/StreamChatUI/Appearance.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public struct Appearance {
+    public var colorPalette = ColorPalette()
+    public var font = Font()
+    public var images = Images()
+    
+    public init() {}
+}
+
+// MARK: - Appearance + Default
+
+private var defaultAppearance: Appearance = .init()
+
+public extension Appearance {
+    static var `default`: Self {
+        get { defaultAppearance }
+        set { defaultAppearance = newValue }
+    }
+}
```

Comment:
I don't think we need this workaround one when appearance is not generic. Can't it be just a normal static variable?